src/main/java/employeetracker/model/employee/UniqueEmployeeList.java

```diff
@@ -163,14 +163,14 @@ public void sortEmployee(String field) {
     }
 
     /**
-     * Get the number of employees in the list.
+     * Gets the number of employees in the list.
      */
-    public int noOfemployees() {
+    public int getNoOfemployees() {
```

Comment:
Perhaps it should be `getNoOfEmployees()` instead of `getNoOfemployees()` to follow Java convention?